apps/meteor/client/views/marketplace/AppsPage/AppsFilters.tsx

```diff
@@ -59,6 +59,8 @@ const AppsFilters = ({
 		private: t('Search_Private_apps'),
 	};
 
+	const fixFiltersSize = window.innerWidth > 1060 ? { maxWidth: 'x200', minWidth: 'x200' } : null;
```

Comment:
This not going to work well if the viewport resizes (e.g. changing window size, opening side panels like DevTools). Prefer to use `useMediaQuery` hook.

Reply:
Thanks for the feedback! :)